### DISCUSSION TITLE
Framework for making games 🎮

### DISCUSSION THREAD

**Discussion by MrKrishnaAgarwal** [scaleracademy/open-source-projects#95] [Ideas]
Currently free open source gaming has a lot of space and opportunities to grow.
There are quite a lot of [free and open source games](https://en.wikipedia.org/wiki/List_of_open-source_video_games), however most of those games have been out for a long time without receiving many updates or improvements.
Since [Godot Engine](https://godotengine.org/) currently has the most active community of all free and open source game engines, I would suggest that it gets used as a base for this game framework project.

Successful creation of such framework would allow for much faster and more efficient creation of new games and it will help beginners to make games in an easy way.

**I hope this idea gets implemented and if you agree with me, do upvote and comment on this discussion.**

**Comment by Dhruvbagadiya** [comment]
It is great idea to making games using framework If there is some inbuilt engines available, which are helpful for making games

**Reply by MrKrishnaAgarwal** [reply]
Yeah, Thanks
